<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="hp-pc" ignored="0" name="Test" tests="3" failures="1" timestamp="2023-07-03T20:46:31 IST" time="47.69" errors="0">
  <testcase name="login" time="1.955" classname="com.TestLayer.Demo1"/>
  <testcase name="validateTitle" time="6.61" classname="com.TestLayer.Demo1"/>
  <testcase name="validateUrl" time="0.012" classname="com.TestLayer.Demo1">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at com.TestLayer.Demo1.validateUrl(Demo1.java:32)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- validateUrl -->
</testsuite> <!-- Test -->
